platform/machine/qemu/machine.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/coreos/mantle/platform"
 	"github.com/coreos/mantle/platform/local"
 	"github.com/coreos/mantle/system/exec"
+
+	"github.com/coreos/pkg/multierror"
```

Comment:
Instead of making another block let's move this up with the import of `golang.org/x/crypto/ssh` similar to https://github.com/coreos/mantle/blob/6b13ddf1ce09e21f6dd0708e571a3bd02418d369/platform/base.go#L25